mesh/nurbs.hpp

```diff
@@ -696,7 +696,7 @@ class NURBSExtension
                   Mode mode = Mode::H_1);
    /// Construct a NURBSExtension by merging a partitioned NURBS mesh.
 
```

Comment:
Looks like the newline is in the wrong place here.

Reply:
I didn't notice this -- it was like this before. I'll move the newline.